### PR TITLE
background_jobs segment : add number of jobs if >1

### PR DESCRIPTION
**Pull request by V1rgul**
I think it should be the default behavior, but I've added `POWERLEVEL9K_BACKGROUND_JOBS_VERBOSE` var to disable it :)

### CODE REVIEW

**Comment by dritter** [comment]
Cool idea! :)

**Comment by bhilburn** [comment]
@V1rgul - I love this! Excellent feature.

Can you add a commit that updates the README.md to document the new verbose variable?

File: powerlevel9k.zsh-theme [hunk 1]
```diff
@@ -290,9 +290,15 @@ prompt_aws() {
 }
 
 # Segment to indicate background jobs with an icon.
+set_default POWERLEVEL9K_BACKGROUND_JOBS_VERBOSE true
 prompt_background_jobs() {
-  if [[ $(jobs -l | wc -l) -gt 0 ]]; then
-    "$1_prompt_segment" "$0" "$2" "$DEFAULT_COLOR" "cyan" "" 'BACKGROUND_JOBS_ICON'
+  local background_jobs_number=$(jobs -l | wc -l)
```
Comment (dritter) [inline powerlevel9k.zsh-theme]: Just found a little glitch: The default `wc` from OSX outputs additional whitespaces.. You could strip them like this:
`local background_jobs_number=${$(jobs -l | wc -l)// /}`

**Comment by V1rgul** [comment]
@bhilburn updated :)

**Comment by bhilburn** [comment]
Great job, @V1rgul!! Thanks for the excellent contribution!

**Pull request merged**